src/js/hooks/cycleCount/useCountStep.jsx

```diff
@@ -123,8 +174,45 @@ const useCountStep = () => {
     setIsStepEditable(true);
   };
 
-  const save = () => {
-    console.log('save');
+  const save = async () => {
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
why is this needed?

Reply:
Looking at this code, I think you can disable those rules for the whole file instead of disabling the same rules x times

Reply:
It's needed because eslint suggested to use `.forEach` method instead of loop, but as you know, we can't use async actions inside `.forEach`